test/rtos_drivers/usb/serial_send_receive.py

```diff
@@ -6,6 +6,7 @@
 import serial
 import serial.tools.list_ports
 import argparse
+import time

```

Comment:
leftover import?

Reply:
Fixed in https://github.com/xmos/fwk_rtos/pull/214/files.